### DISCUSSION TITLE
I am not able to use github copilot even tho i already paid for the yearly subscription.

### DISCUSSION THREAD

**Discussion by ramyatrouny** [community/community#40816] [Copilot Conversations] [answered]
### Github Copilot subscription

Bug

### Not able to use GitHub copilot subscription

I already paid for the GitHub copilot subscription but cannot use it. it keeps on showing that the subscription has ended. 
I have the transaction id and the receipt. can someone support me?

**Comment by Akash1134** [comment]
Hi @ramyatrouny We're sorry that you're facing this issue. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by davecheney** [comment, marked as answer]
@ramyatrouny your cc was declined so your copilot access has been suspended. Please contact support, https://support.github.com/contact.
